### PR TITLE
[Azure Stack] Updates to Azs.Commerce.Admin spec for autorest config

### PR DESCRIPTION
**Pull request by viananth**
### Latest improvements:
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i><br>
### Contribution checklist:
- [ ] I have reviewed the [documentation](https://github.com/Azure/adx-documentation-pr/wiki/Overall-basic-flow) for the workflow.
- [ ] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.
- [ ] The [OpenAPI Hub](https://aka.ms/openapiportal) was used for checking validation status and next steps.
### ARM API Review Checklist
- [ ] Service team MUST add the "WaitForARMFeedback" label if the management plane API changes fall into one of the below categories. 
- adding/removing APIs.
- adding/removing properties.
- adding/removing API-version. 
- adding a new service in Azure.

Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs.
- [ ] If you are blocked on ARM review and want to get the PR merged urgently, please get the ARM oncall for reviews (RP Manifest Approvers team under Azure Resource Manager service) from IcM and reach out to them. 
Please follow the link to find more details on [API review process](https://armwiki.azurewebsites.net/rp_onboarding/ResourceProviderOnboardingAPIRevieworkflow.html).

### CODE REVIEW

File: specification/azsadmin/resource-manager/commerce/readme.azsautogen.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# AutoRest v3 generators
```
Comment (ArcturusZhang) [inline specification/azsadmin/resource-manager/commerce/readme.azsautogen.md:1]: May I know what is this file used for?

Comment (viananth) [reply specification/azsadmin/resource-manager/commerce/readme.azsautogen.md:1]: This is just a config addition for generating the powershell cmdlets using autorest powershell generator.

File: specification/azsadmin/resource-manager/commerce/readme.md [hunk 2]
```diff
@@ -61,27 +61,11 @@ csharp:
...
-require: $(this-folder)/../../../../profiles/readme.md
+> see https://aka.ms/autorest
 
-# all the input files across all versions
+``` yaml
 input-file:
-  - $(this-folder)/Microsoft.Commerce.Admin/preview/2015-06-01-preview/Commerce.json
-
+  - Microsoft.Commerce.Admin/preview/2015-06-01-preview/Commerce.json
 ```
```
Comment (ArcturusZhang) [inline specification/azsadmin/resource-manager/commerce/readme.md:71]: I am not quite sure why this is deleted. This segment should be auto-generated everytime when a PR gets merged. The changes here may be overridden

**Review by ArcturusZhang — COMMENTED** [review]
Since this swagger is under preview therefore the breaking change should be fine here

File: specification/azsadmin/resource-manager/commerce/readme.azsautogen.md (hunk 1 above)
Comment (ArcturusZhang) [reply specification/azsadmin/resource-manager/commerce/readme.azsautogen.md:1]: Got it

**Comment by viananth** [comment]
@ArcturusZhang could you pls approve this PR if it looks good to you? Thanks!

**Review by ArcturusZhang — APPROVED** [review]

**Comment by ArcturusZhang** [comment]
Hi @akning-ms could you please help us merge this PR? This is a preview swagger therefore breaking changes should be fine.

**Pull request merged**